Bootstraps/Symfony.php

```diff
@@ -88,7 +89,7 @@ public function getApplication()
             $app->booted = true;
         }, $app);
 
-        if ($this->debug) {
+        if ($this->debug && ProcessSlave::$slave !== null) {
```

Comment:
Weird side effect, when running the tests we have no Slave process and the `register_file` function fails miserably, not sure if we should fix the function instead...